component-models.json

```diff
@@ -1491,7 +1491,10 @@
         "name": "blogNumberOfArticles",
         "label": "Number Of Articles Display",
         "valueType": "number",
-        "value": 0
+        "value": 13,
+        "validation": {
+          "numberMin": 0
```

Comment:
@mayur-satav Can we change the min number to 3 or more? The 0 will not display the results.

Reply:
updated to 3